test/functional/ui/incsubstitute_spec.lua

```diff
@@ -52,6 +52,31 @@ local function common_setup(screen, inccommand, text)
   end
 end
 
+describe(":substitute, inccommand=split does not trigger preview", function()
+  before_each(function()
+    clear()
+    common_setup(nil, "split", default_text)
+  end)
+
+  it("when invoked by feedkeys() in a script ", function()
+    source(':call feedkeys(":%s/tw/MO/g\\<CR>")')
+    wait()
+    eq(1, eval("bufnr('$')"))
```

Comment:
Are you sure this checks what we want? By putting `\<CR>` at the end of the source, the command would finish, so the preview window is supposed to be closed at this point anyways. Maybe it's useful to check for it, but you can't be sure it didn't show inbetween,  can you?